UniversalAlgebra/Algebras/Congruences.lagda

```diff
@@ -30,7 +30,7 @@ open import Data.Product                          using    ( _,_      ;   Σ
                                                            ; Σ-syntax ;   _×_   )
 open import Relation.Binary.PropositionalEquality using    ( sym ; trans ; cong )
 
-open import Algebras.Products      𝑆  using (ov)
+open import Algebras.Products      {𝑆 = 𝑆}   using (ov)
```

Comment:
Actually, it is just because in a very special circumstance I don't want to mention the signature.  We can change back later, but for now this helps me prepare a monolithic import file that I'm using for a demo/talk I'll give at TYPES.

Reply:
Ah, that context helps explain things. I guess this isn't meant to be merged into mainline?